src/components/qas/index.jsx

```diff
@@ -17,11 +17,11 @@ class QAs extends React.PureComponent {
 
     return (
       <div className="qas">
-        {questions.map(question => {
+        {questions.map((question, qid) => {
           const { question: q, number } = question;
           const primeQ = q[0];
           const { id, questionType } = primeQ;
-          const key = `qa-${id}`;
+          const key = `qa-${id}-${qid}`;
```

Comment:
I didn't notice this last time around but if you're doing this because you're getting an error about unique keys, that's a good error and you should change whatever the duplicate id is instead of adding the index onto the key to make it unique. In other words, it's important for keys to be unique, but for our answers to appropriately line up with our questions, it's also important for the question ids to be unique. Please update.

Reply:
Do you mean to say, update _all the questions_ in the jsons?

Reply:
In this specific case, the Coloring the Universe investigation has a widget and question that gets dragged across multiple pages that needs to persist its answer. How should we handle this, then?

Reply:
shouldn't be a problem since keys' uniqueness is scoped to their loop, not the page.

Reply:
Interesting, but if that was the case why does the app throw the error?